tools/nni_annotation/code_generator.py

```diff
@@ -369,5 +377,8 @@ def parse(code):
         if type(nodes[i]) is ast.ImportFrom and nodes[i].module == '__future__':
             last_future_import = i
     nodes.insert(last_future_import + 1, import_nni)
+    if nas_mode in ['enas_mode', 'oneshot_mode']:
```

Comment:
please add comments here: enas and oneshot modes for tensorflow need tensorflow module, so we import it here.